### DISCUSSION TITLE
Unable to download the latest pre-release

### DISCUSSION THREAD

**Discussion by Gunhedo** [valinet/ExplorerPatcher#3504] [General]
Hi,

I'm getting the following message, when I try to install the latest pre-release.

![image](https://github.com/user-attachments/assets/99d83290-c59d-45df-9693-095725af0b59)

**Comment by Gunhedo** [comment]
I downloaded it manually now and that was working for me.

**Comment by pyrates999** [comment]
> duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3053
> 
> please close this.

Ignore this.

**Comment by Amrsatrio** [comment]
Please read the note in here.

https://github.com/valinet/ExplorerPatcher/releases

**Reply by pyrates999** [reply]
Ohh didn't know that, thanks :)
